scripts/gulp/tasks/test-e2e.ts

```diff
@@ -39,10 +35,14 @@ task('test:e2e:serve:start', async () => {
 task('test:e2e:serve:stop', () => forceClose(server));
 task('test:e2e:serve', series('test:e2e:build', 'test:e2e:serve:start'));
 
-task(
-  'test:e2e:run',
-  cypress({ serverUrl, rootDir: paths.e2eTests(), testNamePattern: argv.testNamePattern as string }),
-);
+task('test:e2e:run', async cb => {
+  const serverHost = 'localhost';
+  const e2ePort = Number(process.env.E2E_PORT) || 8082;
```

Comment:
```suggestion
  // Port should resolve inside a task as it will be set by "test:e2e:serve:start" task
  const e2ePort = Number(process.env.E2E_PORT) || 8082;
```